src/stylesheets/_variables.scss

```diff
@@ -0,0 +1,27 @@
+// Colors
+$brand-black: (
+  light: hsl(240, 11%, 9%),
+  primary: hsl(0, 0%, 0%)
+);
+
+$brand-white: (
+  primary: hsl(0, 0%, 100%)
+);
+
+$brand-gray: (
+  transparent: hsla(0, 0%, 15%, 0.6)
+);
+
+// Font Size
+$font-sizes: (
+  sm: 0.8125rem,
```

Comment:
@noor-ahmed Should we use `rem()` function here also?
If we cannot use it here, we should add some comment after this line to tell how the size of the pixel for this `rem`. 👍